### ISSUE TITLE
Add in Chevy Bolt data

### ISSUE THREAD

**Issue by CCOSTAN** [CCOSTAN/Home-AssistantConfig#297]
https://github.com/home-assistant/home-assistant/pull/11409

Waiting on this to be merged into Prod or at least Dev.

**Comment by CCOSTAN** [comment]
http://danielfrg.com/blog/2015/09/28/crawling-python-selenium-docker/

**Comment by CCOSTAN** [comment]
https://github.com/sdague/mychevy/blob/master/mychevy/mychevy.py#L138-L140

**Comment by CCOSTAN** [comment]
I really want to recreate THIS : 
![image](https://user-images.githubusercontent.com/2160436/36169012-facdbaf0-10c8-11e8-83d5-f1cc093153ad.png)

cc: @sdague

**Comment by CCOSTAN** [comment]
https://github.com/home-assistant/home-assistant/pull/19036/files

Time to put this back in place.

**Comment by CCOSTAN** [comment]
oh and check out the new Energy Assist on CarPlay
![image](https://user-images.githubusercontent.com/2160436/49766488-75bd2900-fca3-11e8-9ed5-dc8e14b07678.png)
